lib/ReactViews/Story/story-builder.scss

```diff
@@ -11,12 +11,13 @@
   transition: all 0.25s;
   transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
   &.isVisible {
+    transition-delay: 0.25s;
```

Comment:
Hi Regina - was this requested by anyone? I think while it successfully mimics the left workbench panel collapsing, it's not necessary for the story as there is no transition to wait for, for it to "connect" like the "Show workbench" button going to the left.
If this isn't by design then let's remove it, it makes the story feel too janky and stuttery when it's just waiting for the close button to appear.

Reply:
I added this because it was a bit of an annoying animation where at some points the close button would overlap with the story button making the animation look bad. Though now that Vic's said
>With the story button, the idea of the ‘X” close button was that we would remove the ‘Story’ button when it is open. However, due to the rounded corners we won’t be doing it this way. So, remove the ‘X’ and just make the 'Story’ button the rollover/active blue button state. If you need more clarification on this pleas speak to Wing.

I think I'll just remove the animation (and the button) entirely and revert it back to what it used to be